weblate/trans/forms.py

```diff
@@ -758,10 +759,21 @@ def get_initial(request: AuthenticatedHttpRequest):
             return {"q": request.GET["q"]}
         return None
 
-    def __init__(self, user: User, language=None, show_builder=True, **kwargs) -> None:
+    def __init__(
+        self,
+        *,
+        request: AuthenticatedHttpRequest,
+        language=None,
```

Comment:
I think we can add type annotation here as well:

```suggestion
        language : Language | None = None,
```